### PR TITLE
Update client statistics codec statistic descriptions

### PR DESCRIPTION
**Pull request by ihsandemir**
Updated client statistics codec javadoc. Described each statistic. Changed "nearcache." to "nc." in the statistics key descriptions.

### CODE REVIEW

File: hazelcast/src/main/java/com/hazelcast/client/impl/protocol/template/ClientMessageTemplate.java [hunk 1]
```diff
@@ -187,56 +187,126 @@ Object authenticationCustom(Data credentials, @Nullable String uuid, @Nullable S
...
-     * runtime.availableProcessors
-     * runtime.freeMemory
-     * runtime.maxMemory
-     * runtime.totalMemory
-     * runtime.uptime
-     * runtime.usedMemory
-     * userExecutor.queueSize
+     * clientType: The string that represents the client type. See {@link com.hazelcast.core.ClientType}
+     *
+     * clusterConnectionTimestamp: The time that the client connected to the cluster (milliseconds since epoch). It is reset on
```
Comment (pveentjer) [inline hazelcast/src/main/java/com/hazelcast/client/impl/protocol/template/ClientMessageTemplate.java]: miliseconds?

File: hazelcast/src/main/java/com/hazelcast/client/impl/protocol/template/ClientMessageTemplate.java [hunk 2]
```diff
@@ -187,56 +187,126 @@ Object authenticationCustom(Data credentials, @Nullable String uuid, @Nullable S
...
+     * {@link com.hazelcast.security.UsernamePasswordCredentials}, this is the username, for custom authentication it is set by
+     * the {@link com.hazelcast.security.Credentials} implementer.
+     *
+     * clientAddress: The address of the client. It is formatted as "<IP>:<port>"
+     *
+     * clientName: The name of the client instance. See ClientConfig.setInstanceName.
+     *
+     * enterprise: "true" if the client is an enterprise client, "false" otherwise.
+     *
+     * lastStatisticsCollectionTime: The time stamp (milliseconds since epoch) when the latest update for the statistics is
```
Comment (pveentjer) [inline hazelcast/src/main/java/com/hazelcast/client/impl/protocol/template/ClientMessageTemplate.java]: miliseconds?

File: hazelcast/src/main/java/com/hazelcast/client/impl/protocol/template/ClientMessageTemplate.java (hunk 1 above)
Comment (ihsandemir) [inline hazelcast/src/main/java/com/hazelcast/client/impl/protocol/template/ClientMessageTemplate.java]: yes, it is System.currentTimeMillis().

File: hazelcast/src/main/java/com/hazelcast/client/impl/protocol/template/ClientMessageTemplate.java (hunk 2 above)
Comment (ihsandemir) [inline hazelcast/src/main/java/com/hazelcast/client/impl/protocol/template/ClientMessageTemplate.java]: yes, it is System.currentTimeMillis().

**Pull request merged**